nixos/modules/misc/version.nix

```diff
@@ -110,6 +112,38 @@ in
       example = "installer";
     };
 
+    image = {
+
+      id = lib.mkOption {
```

Comment:
I'm not sure if I like adding thee fields here without giving them some defaults inferred from `cfg.release` and `cfg.version` etc...  Feels awkward to have to define the values for this manually?

Not sure though

Reply:
i.e. shouldn't we default `IMAGE_ID` to `ID` and `IMAGE_VERSION` to `BUILD_ID` ?

Reply:
Anyhow we can change this later we dont have to block this from being merged on it. Just curious if we can make this nicer out of the box.

Reply:
Mhmm I think this field shouldn't be part of os-release if you're not building an image. I don't believe we should derive these from ID or BUILD_ID. Not a super strong opinion though, just my gut feeling.